test/FIFOMarket.test.ts

```diff
@@ -376,8 +480,146 @@ describe('FIFOMarket', () => {
       });
     });
   });
+
+  describe('when listing supply in the market', () => {
+    it('updates totalActiveSupply, totalNumberActiveRemovals, and activeSupplierCount when a new supplier is added', async () => {
+      const removals = [{ amount: 100 }];
+      const {
+        fifoMarket,
+        totalAmountOfSupply,
+        totalAmountOfSuppliers,
+        totalAmountOfRemovals,
+      } = await setupTestLocal({
+        removalDataToList: removals,
+      });
+
+      const [
+        totalActiveSupply,
+        totalNumberActiveRemovals,
+        activeSupplierCount,
+      ] = await Promise.all([
+        fifoMarket.totalActiveSupply(),
+        fifoMarket.totalNumberActiveRemovals(),
+        fifoMarket.activeSupplierCount(),
+      ]);
+
```

Comment:
I think it would be useful to also expect that `totalAmountOfSupply`, `totalAmountOfRemovals` and `totalAmountOfSuppliers` are expected values since this could return a false positive when `totalActiveSupply` is 0 due to initial values

Reply:
Wrote this
```
      expect(totalAmountOfSupply).to.be.greaterThan(0);
      expect(totalNumberActiveRemovals).to.be.greaterThan(0);
      expect(activeSupplierCount).to.be.greaterThan(0);
```

Is that what you had in mind?

Reply:
that works